### ISSUE TITLE
LXD marketpalce downloader broken

### ISSUE THREAD

**Issue by feldsam** [OpenNebula/one#3554]
**Description**
Some images are not contextualized - centos 7, ubuntu bionic. There is a missing OpenNebula contextualization package.

**To Reproduce**
Download the LXD image from the marketplace.

**Expected behavior**
Contextualization package should be installed during image download.

**Details**
 - Affected Component: Marketplace LXD downloader
 - Hypervisor: LXD
 - Version: 5.8.3

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by feldsam** [comment]
We can test in our dev env. Let me know, if you need help to debug this.

**Comment by dann1** [comment]
Well, the problem is I cannot reproduce this. I just tried centos and bionic containers and they have contextualization package installed.

```
oneadmin@ubuntu1804-lxd-marketplace-89c4f-0:~$ lxc exec one-2 bash
root@ubuntu-bionic---LXD-2:~# apt search one-context
Sorting... Done
Full Text Search... Done
one-context/now 5.8.0-1 all [installed,local]
  Contextualization tools for the virtual machine running in the OpenNebula

root@ubuntu-bionic---LXD-2:~# exit
oneadmin@ubuntu1804-lxd-marketplace-89c4f-0:~$ lxc exec one-3 bash
[root@centos-7---LXD-3 ~]# yum list | grep installed | grep context
Failed to set locale, defaulting to C
one-context.noarch                       5.8.0-1.el7                    installed

```

**Comment by dann1** [comment]
If it is any help. The file in question that processes the images is`/var/lib/one/remotes/datastore/lxd_downloader.sh`, located in the frontend. 

The code that adds stuff inside the images is https://github.com/OpenNebula/one/blob/ad409e4f842b9f75df175b27acab453d0ade3bcc/src/datastore_mad/remotes/lxd_downloader.sh#L152-L205

Maybe you can add debug code and check where things are failing.

**Comment by feldsam** [comment]
Hi, I test too on another infra, and it is working. So this is related to that one env. I have to debug downloader and reply with findings.

**Comment by feldsam** [comment]
Closing this bug, the problem was caused by an unexcepted OpenNebula version number. We build 5.8.3 with some LXD fixes from not released version and named it version 5.8.3-lxdPatch1, which causes a problem in lxd_downloader script and the right context package can not be downloaded.
